src/Components/Assets/AssetTypes.tsx

```diff
@@ -91,6 +91,16 @@ export interface AssetData {
   };
 }
 
+export type AssetUpdate = Partial<AssetData>;
+
+export interface AssetBody {
+  asset: string;
+  bed: string;
+  meta?: {
+    [key: string]: any;
+  };
+}
```

Comment:
This should be `AssetBed` model instead right?

Reply:
Body type in api docs is like this but at other places `AssetBedModel` is used for same body type

```
{
  "asset": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
  "bed": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
  "meta": {
    "additionalProp1": "string",
    "additionalProp2": "string",
    "additionalProp3": "string"
  }
}
```

Reply:
I have left it as AssetBody for now, should I change it?

Reply:
Yes, you can reuse `AssetBedModel` for the same.